Convert printer.message return value to string

julienemo opened the pull request:
**Related issue**

closes #84 

**Changes**

- Converted printer.message return value to string, test updates

- Added ruby 2.7 in travis

- Updated cucumber dependency from 1.2 to 2.4



- [x]  I ran this code locally

* * *

Use
```
gem 'table_print', git: "git://github.com/julienemo/table_print.git", branch: "84_bug_no_method_error_in_ruby_2.7.1"
```
in `Gemfile` til merge

arches commented:
@julienemo thanks - fix looks good... please rebase from master to get the new `.travis.yml` and add ruby 2.7 in there

julienemo commented:
> @julienemo thanks - fix looks good... please rebase from master to get the new `.travis.yml` and add ruby 2.7 in there

Thanks for the update and the reply.

Rebased on master, added 2.7 to travis, updated corresponding test. Working on an error over cucumber (ex: travis job 700304145)

arches commented:
🙏 thanks @julienemo!
